Assets/Content/Data/ApplicationSettings.asset

```diff
@@ -14,10 +14,8 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
   _preload: 1
   _type: 0
+  NetworkType: 2
   Ckey: editorUser
   ServerAddress: 127.0.0.1
-  NetworkType: 2
-  SkipIntro: 1
   EnableDiscord: 0
-  EditorServerAddress: 127.0.0.1
-  EditorServerCkey: editorUser
+  SkipIntro: 1
```

Comment:
I cant tell if this was intentional or was just for testing purposes and accidentally committed.

Reply:
Yeah, I think accidentally pulled to fix the lobby scene. I'll remove it again